Storage/History.swift

```diff
@@ -24,7 +24,7 @@ public protocol BrowserHistory {
     func removeSiteFromTopSites(site: Site) -> Success
 
     func getSitesByFrecencyWithLimit(limit: Int) -> Deferred<Maybe<Cursor<Site>>>
-    func getSitesByFrecencyWithLimit(limit: Int, whereURLContains filter: String) -> Deferred<Maybe<Cursor<Site>>>
+    func getSitesByFrecencyWithLimit(limit: Int, whereURLContains filter: String, includeBookmarked: Bool) -> Deferred<Maybe<Cursor<Site>>>
```

Comment:
I think this name is misleading. We're not including bookmarked items; we're including whether or not the results were bookmarked. Maybe `includeBookmarkState`?